spread.yaml

```diff
@@ -0,0 +1,54 @@
+project: chisel
+
+path: /chisel
+
+backends:
+  # Cannot use LXD backend due to https://github.com/snapcore/spread/issues/154
+  # lxd:
+  #   systems:
+  #     - ubuntu-bionic
+  #     - ubuntu-focal
+  #     - ubuntu-jammy
+  # GitHub actions (runners) don't support nested virtualization (https://github.com/community/community/discussions/8305)
+  # qemu:
+  #   systems:
+  #     - ubuntu-22.04:
+  #         username: ubuntu
+  #         password: ubuntu
+  #     - ubuntu-22.10:
+  #         username: ubuntu
+  #         password: ubuntu
+  adhoc:
+    allocate: |
+      echo "Allocating $SPREAD_SYSTEM..."
+      image=$(echo $SPREAD_SYSTEM | tr '-' ':')
+      docker pull $image
+      docker run -e usr=$SPREAD_SYSTEM_USERNAME -e pass=$SPREAD_SYSTEM_PASSWORD --name $SPREAD_SYSTEM -d $image sh -c '
+        set -x
+        apt update
+        apt install -y openssh-server sudo
+        mkdir /run/sshd /chisel
```

Comment:
The /chisel here should be in prepare as any other ubuntu system won't have it.

Reply:
well `/chisel` is Spread's PATH, so I think it cannot be in "prepare", cause "prepare" runs on the SPREAD_PATH. But it can be removed, cause Spread seems to create SPREAD_PATH if it doesn't exist (tested [here](https://github.com/cjdcordeiro/chisel/actions/runs/3224848928/jobs/5276442232)).